modules/administration-guide/pages/configuring-workspace-target-namespace.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring {orch-namespace} name
+:keywords: administration guide, configuring, namespace
+:page-aliases: 
+
+[id="configuring-user-{orch-namespace}-name-for-automatic-provisioning_{context}"]
+= Configuring user {orch-namespace} name for automatic provisioning
+
+You can configure the {orch-namespace} name template that {prod-short} is using to create the required {orch-namespace} when starting a workspace.
```

Comment:
```suggestion
You can configure the {orch-namespace} name template that {prod-short} uses to create the required {orch-namespace} when starting a workspace.
```
Continuous tense incorrectly implies that the use of the template is a continuous, ongoing process that doesn't end.